x11-misc/synergy_haiku/synergy_haiku-0.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="Mouse and Keyboard sharing software"
+DESCRIPTION="Synergy brings your computers together in one cohesive experience; its free \
+and open source cross-platform software for sharing one mouse and keyboard \
+between multiple computers on your desk. It works on Windows, Mac OS, Linux \
+and Haiku allowing you to seamlessly move your mouse cursor between computers. \
+
+Credits for:
+- Axel Dörfler (ATKeymap)
+- Stefano Ceccherini (wrapper)
+- Jérôme Duval (Keymap)
+- Alex Evans (uSynergy)
+- Jessica Hamilton (wrapper)
+- Ed Robbins (wrapper)
+
+Usage: Create a configuration file at ~/config/settings/synergy with:
+	enable = true
+	server = 192.168.1.X (ip of your server)
+	enableCLipboard = false (false recomended)
```

Comment:
```suggestion
	enableCLipboard = false (false recommended)
```

Fixed grammar. And is the `enableCLipboard` intentional?

Reply:
Yep; its implemented in that way.